.mailmap

```diff
@@ -2,4 +2,5 @@ Nicholas Trienens <nick@fuzzproductions.com> Nicholas Trienens <nick.trienens@mo
 Adam Fowler <adamfowler71@gmail.com> adam-fowler <adam-fowler@users.noreply.github.com>
 Adam Fowler <adamfowler71@gmail.com> dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
 Adam Fowler <adamfowler71@gmail.com> hummingbird-automation[bot] <168462326+hummingbird-automation[bot]@users.noreply.github.com>
-Michael <mredig@gmail.com> Michael <juniper.fife_0b@icloud.com>
\ No newline at end of file
+Michael Redig <juniper.fife_0b@icloud.com>  Michael <mredig@gmail.com>
```

Comment:
Is this your preferred email then?
Run ./scripts/generate-contributors.sh to update the contributors.md

Reply:
It's my hide-my-email address for GitHub. I know it's not doing much given the publicized nature of my other email in commits, etc, but it seems silly to not at least try to obscure it a little bit.

Will do!